code/modules/reagents/chemistry/recipes/pyrotechnics.dm

```diff
@@ -496,3 +496,11 @@
 		return ..()
 	else
 		return FALSE
+
+	/datum/chemical_reaction/reagent_explosion/dimethylmercury
+		name = "Dimethylmercury explosion"
+		id = "dimethylmercury_explosion"
+		mix_message = "<span class='userdanger'>The chemicals vaporize instantly in a rush of heat!</span>"
+		required_reagents = list("dimethylmercury" = 1)
+		required_temp = 278
+		strengthdiv = 1.5 //this IS rocket fuel
```

Comment:
Reagent explosions ignore the maxcap. How big of an explosion does this make in grenade with two bluespace beakers? What about a chembomb? Chembomb with max (7) beakers? What about with fermichem's beaker volume values?

For reference, as of right now, the biggest possible chem explosion via legitimate means is a 12dev explosion, and is only really balanced due to the tedium involved with obtaining a massive supply of corn oil to properly mass produce the chem on top of the tedium of constructing a chembomb. What's the challenge involved in mass producing this chem?

Reply:
Currently: It takes 1/10th of a roundstart chem dispenser's energy to produce 1u of this chem. For 10u it takes almost all 10000 energy. To make a large beaker grenade you need 200u which will take 20 high-cap cells (which are the second highest cap cells behind the 15k engi-exclusive ones.) Once I learn Fermichem I plan to make it MKUltra-levels of tedium to make, as well as dangerous if you mess it up.

Reply:
Also I am aware you can just decon dispenser/ swap cell/ place cell in charger and make chems with fresh cell, and repeat- but that requires a)sourcing another 10k cell and b) a cell charger

Reply:
In my experience from making Fermiexplosions, I'd say have a clamp at a size you expect no one to get to/a size that's unacceptably big, just in case.

Reply:
> To make a large beaker grenade you need 200u

I'm not talking about large beaker grenades. I'm talking about the big boi 'nades that your mother warned you about.

> Energy stuff

Roundstart dispensers only have 1000 energy. Bluespace cells have approximately 4000 energy when no other parts are changed. Bluespace cells with bluespace matter bins bumps it up to 8000 energy. It takes a total of 105 energy to make 1u of this chem. That means for a full 7 beaker chembomb, it'll take 220,500 energy total, and for a simple bluespace beaker grenade, it takes a mere 63,000 energy. That means it takes 28 total bluespace cells to make the biggest boom possible in the game, one that greatly outsizes the supermatter delamination, and a total of 8 bluespace cells for, assuming my math is correct, a 7dev explosion. If my math is also correct, then the biggest possible boom with this chem would be approximately a 13dev explosion. For reference, explosion size is `(2*(chemunits/strengthdiv))^0.5)`, and the devrange is 0.25 that value. If all of my number crunching is correct, then this appears to be fairly balanced, as this chem is only really viable to make as a scientist, though the tedium involved with the setup for efficient production of the chem means that it's only meaningfully useful as either an antag or during clockwork cult. Or maybe mining if the miners are mad enough lads to let you convince them to hypernuke lavaland.

My final verdict is "this is fine as-is"